Applications/Utils/SWMMConverter/SWMMConverter.cpp

```diff
@@ -88,31 +87,42 @@ int main(int argc, char *argv[])
     csv.addVectorForWriting("z", z);
     for (std::size_t i=0; i<9; ++i)
     {
-        std::vector<double> data_vec = swmm->getArrayAtTimeStep(SwmmObject::NODE, 10, i);
-        csv.addVectorForWriting(swmm.getArrayName(SwmmObject::NODE, i), data_vec);
+        std::vector<double> data_vec = swmm->getArrayAtTimeStep(FileIO::SwmmObject::NODE, 10, i);
+        csv.addVectorForWriting(swmm->getArrayName(FileIO::SwmmObject::NODE, i), data_vec);
     }
     csv.writeToFile("d:/csvtest.csv");
     std::cout << "csv written" << std::endl;
     std::cin.ignore();
```

Comment:
I see, this is a nice feature for users.
Can you add a short end-to-end test running as part of the ctests (see for example [tests](https://github.com/ufz/ogs/blob/master/Applications/Utils/Tests.cmake) provided by Tom for the MapGeometryToMeshSurface utility )?